.travis.yml

```diff
@@ -4,10 +4,28 @@ notifications:
   email:
     on_success: never
 
-node_js:
-  - 8
-  - 9
-  - 10
+
+
+jobs:  
+  include:
+    - stage: "JS"
+      node_js: 8
+    - node_js: 9
+    - node_js: 10
+    - stage: "PHP"
+      language: php
+      php: 7.2
+      script:
+        - composer install
+        # PHP syntax check
+        - bash -c '! (find . -name "*.php" ! -path "./vendor/*" -print0 | xargs -0 -n1 -P4 php -l | grep "Parse error")'
```

Comment:
Yes, let's work together to make these tools available for everyone easily, as I plan to use them on native & third party modules.
If you're available to work in a short delay, I'd like to have more insights on your companion. Otherwise, I'd prefer having this PR merged first, as it contains fixes, then refacto these tools in your lib.

Reply:
Should be really better to have a tool with a binary to execute scripts than a repository full of bash scripts :/ I'm really not a big fan of module-companion :/

Reply:
As we deal with PHP projects here, a Composer dependency seems to be a nice fit.

Reply:
:+1:  Coule be cool to have a ./bin/prestashop-cli or something to run our commands :)